PyInstaller/depend/bindepend.py

```diff
@@ -880,3 +880,17 @@ def get_python_library_path():
 
     # Python library NOT found. Return just None.
     return None
+
+def findSystemLibrary(name):
+    '''
+        Given a library name, try to resolve the path to that library. If the
+        path is already an absolute path, return that without searching.
+    '''
+    
+    if os.path.isabs(name):
+        return name
+    
+    if is_unix:
+        return findLibrary(name)
+    elif is_win:
+        return getfullnameof(name)
```

Comment:
Look at func `mac_set_relative_dylib_deps(libname, distname)` in `dylib.py`. It could have some inspiration.